lib/Gearman/Task.pm

```diff
@@ -64,6 +64,15 @@ process.
 
 =item
 
+on_exception
+
+A subroutine reference to be invoked when the task fails. The client
+needs to be configured to support exception handling by setting the
+attribute C<exception> to a true value. The subroutine will be passed
```

Comment:
> attribute `C<exception>`

The attribute name is `exceptions`
https://github.com/p-alik/perl-Gearman/blob/4f221ba9a18dd52de9c54f1fc8731b3e614e743c/lib/Gearman/Client.pm#L71